### PR TITLE
Added load_today and pv_today for 2nd inverter to apps.yaml

### PR DESCRIPTION
**Pull request by gcoan**
If you have two inverters then each inverter operates independently and will separately record its own view of load & PV power.
Changed apps.yaml to include sensors from 2nd inverter

### CODE REVIEW

File: apps/predbat/config/apps.yaml [hunk 1]
```diff
@@ -22,16 +22,20 @@ pred_bat:
   
   #
   # Sensors, currently more than one can be specified and they will be summed up automatically
-  # however if you have two inverters only set one of them as they will both read the same.
+  # If you have two inverters, each inverter will have its own load and PV generation data 
+  # but only set the import and export for one of them as these will both read the same.
```
Comment (springfall2008) [inline apps/predbat/config/apps.yaml]: I'm not sure this is true, the load energy sensor should be a CT clamp and hence cover the whole house, unless you are on 3 phase and you have different inverters for each phase.

For PV it depends, if you have two PV inverters it makes sense but many people don't have PV on both.

File: apps/predbat/config/apps.yaml [hunk 2]
```diff
@@ -226,6 +226,10 @@ pred_bat:
   # Standing charge in pounds, can be set to a sensor or manually entered (e.g. 0.50 is 50p)
   # The default below will pick up the standing charge from the Octopus Plugin
   metric_standing_charge: 're:(sensor.(octopus_energy_|)electricity_[0-9a-z]+_[0-9a-z]+_current_standing_charge)'
+  #
+  # If you don't want to include the standing charge in batpred's cost prediction, set metric_standing_charge to zero.
```
Comment (springfall2008) [inline apps/predbat/config/apps.yaml]: Or just delete the line

Comment (gcoan) [reply apps/predbat/config/apps.yaml]: True, either way, I thought it worthwhile adding an explanation of this as I personally prefer to just focus on the consumption. I can't do anything about the standing charge and it was confusing me to include it

File: apps/predbat/config/apps.yaml (hunk 1 above)
Comment (springfall2008) [reply apps/predbat/config/apps.yaml]: I've added some text about this on main now

File: apps/predbat/config/apps.yaml (hunk 2 above)
Comment (springfall2008) [reply apps/predbat/config/apps.yaml]: I've added some text about this on main now

**Review by springfall2008 — APPROVED** [review]

**Pull request merged**